hasher-matcher-actioner/hmalib/lambdas/api/exchanges.py

```diff
@@ -0,0 +1,92 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+
+"""
+APIs to power viewing and editing of exchanges.
+"""
+
+import bottle
+
+from threatexchange.exchanges import auth
+
+from hmalib.lambdas.api.middleware import SubApp
+from hmalib.aws_secrets import AWSSecrets
+from hmalib.common.config import HMAConfig
+from hmalib.common.configs.tx_apis import (
+    ToggleableSignalExchangeAPIConfig,
+    add_signal_exchange_api,
+    set_status_signal_exchange_api,
+)
+
+
+def get_exchanges_api(hma_config_table: str, secrets_prefix: str) -> bottle.Bottle:
+    exchanges_api = SubApp()
+    HMAConfig.initialize(hma_config_table)
+    secrets = AWSSecrets(secrets_prefix)
+
+    @exchanges_api.get("/")
+    def get_exchanges():
+        """
+        Get the currently enabled APIs.
```

Comment:
blocking: Seems like you are returning them all, not just the enabled ones (update comment)